check-hana-cloud-audit-logs/catalog.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
@kloyan , the command works as a charm. I was thinking on extending it by adding output results so that the user can see if there are any entries out of the check. 
In example, something like this one: 
```
results --> string
$(.ReadAuditLogs.output.result)
```